### DISCUSSION TITLE
JPA Auditing 기능

### DISCUSSION THREAD

**Discussion by YoonJae00** [finalprojectSinchon/backend#10] [Ideas]
![Auditing](https://github.com/user-attachments/assets/ab3d7171-fada-4183-9a3e-3503208619c2)

![스크린샷 2024-07-14 오후 4 43 34](https://github.com/user-attachments/assets/30e3612b-4de2-4b9e-bf1d-961d405752e5)

Entity 에 상속만 해주면 알아서 들어옴

![스크린샷 2024-07-14 오후 4 44 12](https://github.com/user-attachments/assets/47b85356-ac8a-4a0a-921d-7603a574a625)

**Comment by rjadmsehf** [comment]
좋은 정보 감사합니다. !! ~~ ^.^

**Comment by rraaccoon** [comment]
<strong>훌륭합니다.</strong> 이렇게 계속 스토리를 쌓아보세요~~👍
